graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH6.kt

```diff
@@ -0,0 +1,142 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.HasCycle, GraphBehaviours.CreateDirectedGraph, """
+    Determine if one or more cycles exist in graph
+""")
+class BEH6 : GraphScenario() {
+
+    @Eac("When a directed graph has one cycle, return true")
```

Comment:
🚫 This reads like a unit test and I don't feel that it is capturing the essence of the requirement in its generality.
```suggestion
    @Eac("When a directed graph has at least one cycle, return true")
```